src/components/LiquidityMining/Create/Steps/PreviewAndCreate/SimulateStaking/index.tsx

```diff
@@ -0,0 +1,229 @@
+  setSimulatedStakedAmount: (value: string) => void
+  setSimulatedPrice: (value: any) => void
+  simulatedPrice: any
+  loading: boolean
+}
+enum SimulateOptions {
+  AMOUNT = 'Amount',
+  PRICE = 'Price',
+}
+
+export default function SimulateStaking({
+  tokenOrPair,
+  stakingCap,
+  setSimulatedStakedAmount,
+  setSimulatedPrice,
+  simulatedPrice,
+  nativeCurrencyUSDPrice,
+  loading,
+}: RewardSummaryProps) {
+  const { loading: loadingNativeTokenPrice, derivedNativeCurrency: nativeTokenPrice } = useTokenOrPairNativeCurrency(
+    tokenOrPair ? tokenOrPair : undefined
+  )
+
+  const handleLocalStakingCapChange = useCallback(
+    rawValue => {
+      setSimulatedPrice(rawValue)
+    },
+    [setSimulatedPrice]
+  )
+  useEffect(() => {
+    if (nativeTokenPrice && nativeCurrencyUSDPrice)
+      setSimulatedPrice(nativeTokenPrice.multiply(nativeCurrencyUSDPrice).toFixed(2))
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [loadingNativeTokenPrice, nativeTokenPrice])
+
+  const [simulateOption, setSimulateOption] = useState<SimulateOptions>(SimulateOptions.AMOUNT)
+  const [showUSDValue, setShowUSDValue] = useState(true)
+  const [simulatedValuePercentage, setSimulatedValuePercentage] = useState(10)
+  const liquidityPercentChangeCallback = useCallback((value: number) => {
+    setSimulatedValuePercentage(value)
+  }, [])
+  const [innerLiquidityPercentage, setInnerLiquidityPercentage] = useDebouncedChangeHandler(
+    simulatedValuePercentage,
+    liquidityPercentChangeCallback,
+    10
+  )
+  const maxStakedSimulatedAmount = useMemo(() => {
+    const simulatedPriceAdjusted = simulatedPrice || '0'
+
+    const base = stakingCap
+      ? parseFloat(stakingCap.multiply(simulatedPriceAdjusted).toSignificant(22))
+      : DOLLAR_AMOUNT_MAX_SIMULATION
+
+    const baseInUsd = parseFloat(simulatedPriceAdjusted)
+
+    const baseValue = showUSDValue ? base : base / baseInUsd
+
+    const baseCurrency = tokenOrPair instanceof Token ? tokenOrPair : tokenOrPair?.liquidityToken
+
+    if (baseCurrency && base !== 0 && baseInUsd !== 0) {
+      setSimulatedStakedAmount(
+        parseUnits(
+          calculatePercentage(base / baseInUsd, simulatedValuePercentage).toString(),
+          baseCurrency.decimals
+        ).toString()
+      )
+    }
+
+    return calculatePercentage(baseValue, simulatedValuePercentage)
+  }, [setSimulatedStakedAmount, tokenOrPair, simulatedPrice, simulatedValuePercentage, stakingCap, showUSDValue])
+
+  const handleUSDValueClick = useCallback(() => {
+    setShowUSDValue(!showUSDValue)
+  }, [showUSDValue])
+
+  const handleResetClick = useCallback(() => {
+    setSimulatedValuePercentage(10)
+    setSimulatedPrice(
+      nativeTokenPrice && nativeCurrencyUSDPrice ? nativeTokenPrice.multiply(nativeCurrencyUSDPrice).toFixed(2) : '0'
+    )
+  }, [nativeCurrencyUSDPrice, nativeTokenPrice, setSimulatedPrice])
+
+  return (
+    <SmoothGradientCard
+      justifyContent={'space-between !important'}
```

Comment:
should avoid `{}` when we pass string.
```suggestion
      <SmoothGradientCard
      justifyContent='space-between !important'
      flexDirection='column'
      alignItems='center'
      padding='18px 28px'
      height='162px'
      width='354px'
      >
```
Also it repeats bellow e.g 
```js
<Flex width={'100%'} padding={'10px 12px'} justifyContent={'space-between'}>
```

Reply:
Also, we should avoid using `!important` at all costs.

Reply:
yeah bad habit!